app/views/pages/report.ejs

```diff
@@ -194,12 +194,15 @@
       <label class="mr-4" for="gender-female"><%= __("Female") %></label>
 
       <input required type="radio" id="gender-male" name="gender" value="male" <%= sex === 'MALE' ? 'checked' : '' %>>
-      <label for="gender-male"><%= __("Male") %></label>
+      <label class="mr-4" for="gender-male"><%= __("Male") %></label>
+
+      <input required type="radio" id="gender-x" name="gender" value="x" <%= sex === 'X' ? 'checked' : '' %>>
+      <label for="gender-x"><%= __("X") %></label>
     </p>
     </fieldset>
 
     <label for="postal-code"><%= __('Zip code') %></label>
-    <input class="appearance-none border rounded py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" required type="text" pattern="[0-9]{<%= locals.zipLength %>}" maxlength="<%= locals.zipLength %>" name="postal-code" id="postal-code" placeholder="<%= locals.zipPlaceHolder %>" value="<%= postalCode %>">
+    <input class="appearance-none border rounded py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" required type="text" pattern="[A-Za-z][0-9][A-Za-z]" maxlength="<%= locals.zipLength %>" name="postal-code" id="postal-code" placeholder="<%= locals.zipPlaceHolder %>" value="<%= postalCode %>">
```

Comment:
I suspect this change is going to break some existing languages. We have a couple of options:
1. Remove the pattern validation (don't know how much it helps)
2. Move the pattern to the config
3. Use a very wide pattern (`[A-Za-z0-9]{3,<%= locals.zipLength %>}`)

Reply:
I used option 3.